test/stubs.coffee

```diff
@@ -151,6 +152,7 @@ beforeEach ->
   @stubs.chatMock =
     postMessage: (msg, room, opts) =>
       @stubs.send(msg, room, opts)
+      Promise.resolve()
   @stubs.conversationsMock =
     setTopic: (id, topic) =>
```

Comment:
does this stubbed method need a `Promise.resolve()`?

Reply:
yeah it was failing a test without it because there's some code that uses .catch() on the result of this method.

Reply:
and unlike `SlackBot#send()`, this actually does have an declared return type, so we'd want the stub to have the same.